int32 motor_set_position_positive
int32 motor_set_position_negative
int32 motor_set_position_absolute
bool motor_go_home

int32 linear_set_position_positive
int32 linear_set_position_negative
int32 linear_set_position_absolute

uint16 servo1_target
uint16 servo2_target

bool probe_take_reading

bool overtravel
